packages/wallet-management/src/hooks/useCombinedWallets.ts

```diff
@@ -97,9 +107,11 @@ export const useCombinedWallets = () => {
   const walletConfig = useWalletManagementConfig()
   const bigmiConfig = useBigmiConfig()
   const wagmiAccount = useAccount()
-  const bigmiAccount = useAccount({ config: bigmiConfig })
+  const bigmiAccount = useBigmiAccount({ config: bigmiConfig })
```

Comment:
We can simplify this and change it to just

`const bigmiAccount = useBigmiAccount()`

and also remove 

`const bigmiConfig = useBigmiConfig()`

since we now have Bigmi own config context. Same goes for `useBigmiConnect` below.

Reply:
yeah, make sense, thanks for this.